common/src/main/java/com/wynntils/services/translation/TranslationService.java

```diff
@@ -68,9 +77,19 @@ public void resetTranslator() {
         translator = null;
     }
 
+    private void addShutdownHook() {
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            CachingTranslationProvider.saveTranslationCache();
+            WynntilsMod.info("Successfully saved translation cache!");
+        }));
+    }
+
     public enum TranslationServices {
-        GOOGLEAPI(GoogleApiTranslationProvider.class),
-        PIGLATIN(PigLatinTranslationProvider.class);
+        GOOGLEAPI(GoogleApiTranslationProvider.class);
+
+        // This is a demo service, not used in production.
+        // Users accidentally enabling this service could cause confusion and frustration.
+        // PIGLATIN(PigLatinTranslationProvider.class);
```

Comment:
Unfortunately this will make IntelliJ complain that`PigLatinTranslationProvider` is unused, but maybe we can live with that.

Reply:
I did not want to remove the code, but unfortunately, we can't have conditional enums.